### PR TITLE
fixed build errors when DHT with SNMP protocol is enabled

### PR DESCRIPTION
**Pull request by premultiply**
fixed pull request by Andreas Bleischwitz

### CODE REVIEW

File: protocols/snmp/snmp.c [hunk 1]
```diff
@@ -287,37 +287,46 @@ tank_next(uint8_t * ptr, struct snmp_varbinding * bind)
 uint8_t
 dht_polling_delay_reaction(uint8_t * ptr, struct snmp_varbinding * bind, void *userdata)
 {
-  if (bind->len == 1 && bind->data[0] == 0) {
-    return encode_short(ptr, SNMP_TYPE_INTEGER, dht_global.polling_delay);
-  } else {
+  if (bind->len != 1 && bind->data[0] >= dht_sensors_count) {
```
Comment (eku) [inline protocols/snmp/snmp.c]: && should be II

Why did you add two empty lines to each if-block?

**Comment by premultiply** [comment]
Should be fixed now.
